bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/TimeAmount.scala

```diff
@@ -13,7 +16,9 @@ object TimeAmount {
   // with the difference that we try to keep the time unit from changing if they
   // are all the same.
   def sum(times: Traversable[TimeAmount]): TimeAmount =
-    if (times.isEmpty) empty else (times.head/:times.tail)(_ + _)
+    if (times.isEmpty) empty else (times.head/:times.tail)(_ |+| _)
```

Comment:
Do we really need `Traversable` here? `TimeAmount.sum(ts)` can normally be written `ts.suml`. In any case this method should just be `foldLeft` starting with `empty`; the `if` is unnecessary.